components/atom/progressBar/src/index.scss

```diff
@@ -0,0 +1,42 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$bg-progress-bar: $c-gray-lightest;
+$bc-progress-bar: $c-gray-light;
+$c-progress-bar-fill: $c-primary;
+
+$h-progress-bar: $m-base;
+
+.sui-AtomProgressBar {
+  &-indicator {
+    display: inline-block;
+    font-size: $fz-s;
+    line-height: $lh-s;
+
+    &--top {
+      margin-bottom: $m-base;
+    }
+
+    &--bottom {
+      margin-top: $m-base;
+    }
+    
+  }
+
+  &-container {
+    background: $bg-progress-bar;
+    border: 1px solid $bc-progress-bar;
+    border-radius: 25px;
```

Comment:
As the 25px is used twice, might be is a good idea to extract to a variable in order to change once and get the value. Even if not customizable.

Reply:
👍